docs/source/getting-started/installation.rst

```diff
@@ -66,6 +68,8 @@ To get the **AWS access key** required by :code:`aws configure`, please go to th
 
 To use AWS IAM Identity Center (AWS SSO), see :ref:`here<aws-sso>` for instructions.
 
+[**Optional**] To create a new AWS user with minimal permissions for SkyPilot, see :ref:`AWS User Creation <cloud-permissions-aws>`.
```

Comment:
Nit: Can we `[**Optional**]` -> `**Optional**:`? Looks better to me. Same for GCP below?

Reply:
Sounds good. Updated.